numpy/distutils/command/build_ext.py

```diff
@@ -120,7 +120,7 @@ def run(self):
         self.compiler.show_customization()
 
         # Setup directory for storing generated extra DLL files on Windows
-        self.extra_dll_dir = os.path.join(self.build_temp, 'extra-dll')
+        self.extra_dll_dir = os.path.join(self.build_temp, '.libs')
```

Comment:
Is it necessary to change the folder name as part of this fix? I know of at least one library that has `extra-dll` hard-coded in its own `__config__.py`-like script. Keeping the old name would not break those cases.

Reply:
Sorry for not getting this right the first time, but this should have been named `.libs` to begin with as  auditwheel does because it serves the same function.

Reply:
If this change goes in, it needs a compatibility entry in the release notes.

Reply:
@letmaik What library is that? It worries me that such things are out there.

Reply:
This section seems new in 1.14, so there is still time time to change it. @pv thoughts?

Reply:
As far as scipy is concerned, this doesn't matter and does not introduce extra work. I'm not sure if trying to match auditwheel is a strong argument as the name is quite arbitrary (as long as it is not a valid name for a subpackage), but maybe using same name makes it easier for downstream repackagers (pyinstaller?) etc.

Reply:
This is an implementation detail that should not be relied on because `__config__.py` is the public API. Perhaps this should be clarified in the docs but this was not done in SciPy and it therefore has no effect on SciPy.

Reply:
Thinking long term it may be possible to push this into Python itself but that is not possible currently because extra-dll is a valid package name.

Reply:
There seems to be quite a bit out there: https://github.com/search?q=extra-dll&type=Code&utf8=%E2%9C%93. Ignore the `NCAR/wrf-python` repo, this was a fix brought in by myself and I can change it easily if necessary.

Reply:
Doesn't matter because this was not yet released. We  cannot guarantee API stability in the development branch -- that's unreasonable.

Reply:
OK nevermind, didn't realize this was a new thing, go for it then. :)